ui-admin/src/user/UserProvider.tsx

```diff
@@ -9,15 +9,17 @@ export type UserContextT = {
   user: AdminUser | null,
   loginUser: (adminUser: AdminUser) => void,
   loginUserUnauthed: (adminUser: AdminUser) => void,
-  logoutUser: () => void
+  logoutUser: () => void,
+  toggleSuperuserOverride: () => void
 }
 
 /** current user object context */
 export const UserContext = React.createContext<UserContextT>({
   user: null,
   loginUser: () => { throw new Error('context not yet initialized') },
   loginUserUnauthed: () => { throw new Error('context not yet initialized') },
-  logoutUser: () =>  { throw new Error('context not yet initialized') }
+  logoutUser: () =>  { throw new Error('context not yet initialized') },
+  toggleSuperuserOverride: () => { throw new Error('context not yet initialized') }
 })
```

Comment:
we have a util for this (that should also have been used above)
```suggestion
  logoutUser: emptyContextAlertFunction,
  toggleSuperuserOverride: emptyContextAlertFunction
})
```